### ISSUE TITLE
functor(T,F,N) T-structure syntax error undetected

### ISSUE THREAD

**Issue by flexoron** [trealla-prolog/trealla#582]
v1.29.15
```
?- T = f\\(\\).
Error: syntax error, operator expected unfinished input '.', line 1
   false.

?- functor(f\\(\\),F,N).
   false.
```

**Comment by UWN** [comment]
```
?- current_op(Pri,Fix,\\).
   false.
?- functor(f\\(\\),F,N).
   false, unexpected.
?- X=[functor(f\\(\\),F,N)].
   X = [functor(f,\\(\\),F)], unexpected.
```

**Comment by flexoron** [comment]
oh, thanx @UWN
```
?- op(100,xfx,\\).
   true.
?- functor((\\) \\ \\ \\,F,N).
   F = 2.
```

**Comment by UWN** [comment]
It would help a lot, if Trealla would concentrate on strict ISO conformity. Instead, one intended extension after the other is made such that systematic testing becomes impossible. Just think of [`number_chars/2`](https://github.com/trealla-prolog/trealla/issues/253).

**Comment by flexoron** [comment]
v1.29.18, some issues fixed.
open:
```
?- op(100,xfx,\\).
   true.
?- functor((\\) \\ \\ \\,F,N).
   F = 2.  % unexpected
```

**Comment by infradig** [comment]
What's the status here? Seems ok now.

**Comment by flexoron** [comment]
No (v1.33.12)
```
?- op(100,xfx,\\).
   true.

?- X = [functor((\\) \\ \123,F,N)].
   X = [functor((\\)\\(\),123,F)].
```

**Comment by flexoron** [comment]
Fixed! (v1.33.14)
```
?- X = [functor((\\) \\ \123,F,N)].
Error: syntax error, operator clash, line 1
   true.
?- X = [functor((\\) \\ (\123),F,N)].
   X = [functor((\\)\\(\123),F,N)].
?- functor((\\)\\(\123),F,N).
   F = (\\), N = 2.
```
